webui/src/pages/auth/groups/group/members.jsx

```diff
@@ -26,15 +26,41 @@ const GroupMemberList = ({ groupId, after, onPaginate }) => {
     const [refresh, setRefresh] = useState(false);
     const [showAddModal, setShowAddModal] = useState(false);
     const [attachError, setAttachError] = useState(null);
-
+    const [allUsers, setAllUsers] = useState([]);
     const {results, loading, error, nextPage} = useAPIWithPagination(() => {
         return auth.listGroupMembers(groupId, after);
     }, [groupId, after, refresh]);
-
     useEffect(() => {
         setAttachError(null);
     }, [refresh]);
 
+    const setAllUsersFromLakeFS = async () => {
```

Comment:
This is not only setting the users, but mainly fetching them.
This is important, since when I'm calling this function, it should be clear that it gets data from the API.

So please rename to `AllUsersFromLakeFS`, or something else that will make it clear that this is about API calls.